poleno/mail/tests/test_mandrill.py

```diff
@@ -303,6 +301,12 @@ class WebhookViewTest(MailTestCaseMixin, ViewTestCaseMixin, TestCase):
 
     urls = u'poleno.mail.transports.mandrill.urls'
 
+    def setUp(self):
+        self.settings_override = override_settings(
+            TEMPLATE_LOADERS=(u'django.template.loaders.filesystem.Loader',),
+        )
+        self.settings_override.enable()
```

Comment:
@viliambalaz Netreba aj `tearDown` s `settings_override.disable()`?

Reply:
Nie netreba, `settings_override.disable()` je volany v rodicovskej classe `MailTestCaseMixin._post_teardown`. Bolo to kvoli testu `test_allowed_http_methods`. :ok: prepisal som to tak.